lib/mayaUsd/fileio/shading/symmetricShaderReader.h

```diff
@@ -48,14 +48,14 @@ PXR_NAMESPACE_OPEN_SCOPE
 /// \code
 /// TF_REGISTRY_FUNCTION(UsdMayaShaderReaderRegistry)
 /// {
-///     PxrUsdTranslators_SymmetricShaderReader::RegisterReader(
+///     UsdMayaSymmetricShaderReader::RegisterReader(
 ///         TfToken("MyUsdShaderId"),
 ///         TfToken("myMayaNodeTypeName"),
 ///         /* shadingConversionName = */ TfToken("myShadingConversionName"));
 /// };
 /// \endcode
 ///
-class PxrUsdTranslators_SymmetricShaderReader : public UsdMayaShaderReader
+class UsdMayaSymmetricShaderReader : public UsdMayaShaderReader
```

Comment:
That file is missing a lot of "MAYAUSD_CORE_PUBLIC" for the DLL exports on windows.

Reply:
Yup, just saw that in the failed preflight run. Fixing that up now. Thanks!

Reply:
Should be addressed in https://github.com/Autodesk/maya-usd/pull/803/commits/6cd7599a2e4ef525f50766ad1ed161f84d62de2b. I kicked off another preflight to confirm.